api/metrics/src/main/java/io/opentelemetry/api/metrics/BoundDoubleCounter.java

```diff
@@ -5,20 +5,28 @@
 
 package io.opentelemetry.api.metrics;
 
+import io.opentelemetry.context.Context;
 import javax.annotation.concurrent.ThreadSafe;
 
-/** A {@code Bound Instrument} for a {@link DoubleCounter}. */
+/** A counter instrument that records {@code double} values with preassociated attributes. */
 @ThreadSafe
-public interface BoundDoubleCounter extends BoundSynchronousInstrument {
+public interface BoundDoubleCounter {
   /**
-   * Adds the given {@code increment} to the current value. The values cannot be negative.
+   * Reecord a value with prebound attributes.
    *
-   * <p>The value added is associated with the current {@code Context}.
+   * <p>Note: This may use {@code Context.current()} to pull the context associated with this
+   * measurement.
    *
-   * @param increment the value to add.
+   * @param value The increment amount. MUST be non-negative.
    */
-  void add(double increment);
+  void add(double value);
+  /**
```

Comment:
Empty line between methods. Seems to be a few spots in the PR

Reply:
should be fixed.